### PR TITLE
Update hotspot-operator.mdx

### PR DESCRIPTION
**Pull request by waveform06**
Clarified Premine rules added note about claiming services.

### CODE REVIEW

File: docs/solana/migration/hotspot-operator.mdx [hunk 1]
```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
...
-- Transferred data.
+- Witnessed and was a valid witness
+- Beaconed and had valid witnesses
+- Transferred LoRaWAN data and been paid for it
 
 A Hotpsot is not "Active" if:
 
-- Beaconed with no valid/invalid witnesses, and
-- Has not passed data
+- Beaconed with no witnesses
```
Comment (jthiller) [inline docs/solana/migration/hotspot-operator.mdx]:
```suggestion
```
Beaconing with no witnesses is encapsulated in beaconing with no valid witnesses.

File: docs/solana/migration/hotspot-operator.mdx [hunk 2]
```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
...
+- Beaconed with no valid witnesses
+- Witnessed but as an invalid witness
```
Comment (jthiller) [inline docs/solana/migration/hotspot-operator.mdx]:
```suggestion
- Has not been a valid witness to a beacon
```

File: docs/solana/migration/hotspot-operator.mdx [hunk 3]
```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
...
+- Witnessed and was a valid witness
+- Beaconed and had valid witnesses
+- Transferred LoRaWAN data and been paid for it
 
 A Hotpsot is not "Active" if:
 
-- Beaconed with no valid/invalid witnesses, and
-- Has not passed data
+- Beaconed with no witnesses
+- Beaconed with no valid witnesses
```
Comment (jthiller) [inline docs/solana/migration/hotspot-operator.mdx]:
```suggestion
- Has not beaconed with valid witnesses
```

File: docs/solana/migration/hotspot-operator.mdx [hunk 4]
```diff
@@ -99,14 +100,16 @@ billion tokens, distributed to all active Hotspots at the time of the Snapshot.
...
 following:
 
-- Witnessed and been valid/invalid witnesses.
-- Beaconed and had valid/invalid witnesses.
-- Transferred data.
+- Witnessed and was a valid witness
+- Beaconed and had valid witnesses
+- Transferred LoRaWAN data and been paid for it
 
 A Hotpsot is not "Active" if:
```
Comment (jthiller) [inline docs/solana/migration/hotspot-operator.mdx]: I could see someone being confused if this list is "any" of these conditions or "all" of these conditions.

Comment (jthiller) [inline docs/solana/migration/hotspot-operator.mdx]:
```suggestion
A Hotspot is not "Active" if it:
```

**Review by samgutentag — APPROVED** [review]

**Pull request merged**